build.sbt

```diff
@@ -13,7 +13,10 @@ lazy val microservice = Project(appName, file("."))
   .settings(resolvers += Resolver.jcenterRepo)
   .settings(publishingSettings: _*)
   .configs(IntegrationTest)
-  .settings(integrationTestSettings(): _*)
+  .settings(inConfig(IntegrationTest)(itSettings): _*)
+  .settings(
+    javaOptions in IntegrationTest += "-Dlogger.resource=logback-it.xml"
```

Comment:
We can group this with the integration test settings below

Reply:
Of course I bloomin can. Good spot @amaalali 👍